de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5556970_sys_gh6471_InsertModelValidatorAndFixCoProductComponentType.sql

```diff
@@ -0,0 +1,9 @@
+-- 2020-04-10T12:13:58.823Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_ModelValidator (AD_Client_ID,AD_ModelValidator_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,ModelValidationClass,Name,SeqNo,Updated,UpdatedBy) VALUES (0,540124,0,TO_TIMESTAMP('2020-04-10 15:13:58','YYYY-MM-DD HH24:MI:SS'),100,'EE01','Y','org.eevolution.callout.PP_Product_BomModuleActivator','Module Activator for PP_Product_Bom',0,TO_TIMESTAMP('2020-04-10 15:13:58','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
* There is already a "main" validator: `org.eevolution.model.LiberoValidator` and it also registers a model intetrceptor: `org.eevolution.model.validator.PP_Product_BOM`.
* you can register the (field-)callout and the tabCallout in its constructor (`Services.get(ITabCalloutFactory.class)`)